scripts/us_bea/states_gdp/states_gdp.tmcf

```diff
@@ -0,0 +1,17 @@
+Node: E:states_gdp->E1
+typeOf: dcs:StatVarObservation
+variableMeasured: dcs:InflationAdjustedGDP
+observationAbout: C:states_gdp->GeoId
+observationDate: C:states_gdp->Quarter
+observationPeriod: "P3M"
+value: C:states_gdp->millions_of_chained_2012_dollars
+unit: "Millions of chained 2012 dollars"
```

Comment:
I think unit here should simply by USDollar, and the value should be expanded out.
I'm not sure of the best way to model the fact that they're adjusted dollars though, I see that we have these already in use as units:

ChainedUSDollarBaseYr2012
ConstantPriceUSDollar2015
AnnualUSDollars
USDollar

So perhaps the first option works best. Here's an example node:
https://browser.datacommons.org/kg?dcid=dc/o/d2b7pv31bwls8

Reply:
Will transition to measuring things directly as USDs, by multiplying all values by 1,000,000. Let me know if this causes any issues.